webpack/react_app/components/RecentJobsCard/RecentJobsTable.js

```diff
@@ -33,55 +25,50 @@ const RecentJobsTable = ({ status, hostId }) => {
   } = useAPI('get', jobsUrl, RECENT_JOBS_KEY);
 
   return (
-    <DataList aria-label="recent-jobs-table" isCompact>
-      <SkeletonLoader
-        skeletonProps={{ count: 3 }}
-        status={responseStatus || STATUS.PENDING}
-        emptyState={
-          <Bullseye>
-            <Text style={{ marginTop: '20px' }} component="p">
-              {__('No results found')}
-            </Text>
-          </Bullseye>
-        }
+    <SkeletonLoader
+      skeletonProps={{ count: 3 }}
+      status={responseStatus || STATUS.PENDING}
+      emptyState={
+        <Bullseye>
+          <Text style={{ marginTop: '20px' }} component="p">
+            {__('No results found')}
+          </Text>
+        </Bullseye>
+      }
+    >
+      <TableComposable
+        aria-label="recent-jobs-table"
+        variant="compact"
+        borders="compactBorderless"
       >
-        {jobs?.length &&
-          jobs.map(
-            ({
-              status: jobStatus,
-              status_label: label,
-              id,
-              start_at: startAt,
-              description,
-            }) => (
-              <DataListItem key={id}>
-                <DataListItemRow>
-                  <DataListItemCells
-                    dataListCells={[
-                      <DataListCell
-                        wrapModifier={DataListWrapModifier.truncate}
-                        key={`name-${id}`}
-                      >
-                        <a href={foremanUrl(`/job_invocations/${id}`)}>
-                          {description}
-                        </a>
-                      </DataListCell>,
-                      <DataListCell key={`date-${id}`}>
-                        <RelativeDateTime date={startAt} />
-                      </DataListCell>,
-                      <DataListCell key={`status-${id}`}>
-                        <JobStatusIcon status={jobStatus}>
-                          {label}
-                        </JobStatusIcon>
-                      </DataListCell>,
-                    ]}
-                  />
-                </DataListItemRow>
-              </DataListItem>
-            )
-          )}
-      </SkeletonLoader>
-    </DataList>
+        <Tbody>
+          {jobs?.length &&
```

Comment:
If there are no jobs, the card just says `0`
![image](https://user-images.githubusercontent.com/7326770/187858013-811e68c5-74b0-4797-bcc9-468ef193efe1.png)

Maybe `map`-ing over an empty array should be valid